utils/runtime-fuzzer/src/runtime/mod.rs

```diff
@@ -45,7 +47,10 @@ pub fn new_test_ext() -> TestExternalities {
     let authorities = vec![authority_keys_from_seed("Authority")];
     // Vector of tuples of accounts and their balances
     let balances = vec![
-        (account(account::alice()), account::acc_max_balance()),
+        (
+            account(account::alice()),
+            block_gas_cost().saturating_mul(20),
```

Comment:
magic nums without any comments

Reply:
_fuzzer related comments_
resolving, a new PR for fuzzer changes will be opened.